FrameworkSystem/scripts/dirac-populate-component-db.py

```diff
@@ -68,6 +71,10 @@ def setExcludedHosts( value ):
 if not resultInfo[ 'OK' ]:
   gLogger.error( resultInfo[ 'Message' ] )
   DIRACexit( -1 )
+resultMySQL = client.getMySQLStatus()
+if not resultMySQL[ 'OK' ]:
+  gLogger.error( resultMySQL[ 'Message' ] )
+  DIRACexit( -1 )
```

Comment:
This is the SystemAdministratorIntegrator call, it will return a list with the result of calling 'getMySQLStatus' on every not excluded host (some hosts will fail and some will be successful). Being this the case, i think the script should just abort if the Integrator has failed during the call, as this information is used later in the script.

The individual results for 'getMySQLStatus' returned by the SysAdminIntegrator are checked later in the loop and will only keep the script from trying to communicate with MySQL in those hosts that don't have it installed.